src/Enum.php

```diff
@@ -229,7 +229,12 @@ public static function search($value)
      */
     public static function label($typeValue): string
     {
-        $langId = 'typelabels.' . get_called_class() . '.' . strtolower(self::search($typeValue));
+        $typeKey = self::search($typeValue);
+        if (! is_string($typeKey)) {
+            $typeKey = (string) $typeKey;
+        }
+
+        $langId = 'typelabels.' . get_called_class() . '.' . strtolower($typeKey);
```

Comment:
This is basically the same right? 

```suggestion
        $langId = 'typelabels.' . get_called_class() . '.' . strtolower((string) self::search($typeValue));
```